.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ executors:
   # Whenever the Go version is updated here, .promu.yml should also be updated.
   golang:
     docker:
-      - image: cimg/go:1.20
+      - image: cimg/go:1.21
```

Comment:
We can bring this all the way up to 1.23.

Reply:
Okay. I did that to .promu.yml too as the comment says to.